test/recipes/80-test_cms.t

```diff
@@ -65,6 +66,8 @@ unless ($no_fips) {
     capture => 1, statusvar => \$dsaallow);
     $no_dsa = 1 if $dsaallow == '0';
     $old_fips = 1 if $dsaallow != '0';
+    run(test(["fips_version_test", "-config", $provconf, "<3.5.0"]),
+    capture => 1, statusvar => \$no_pqc);
```

Comment:
Indentation could be better...

Reply:
I've just copied what was above, but yeah, fixed now. Please re-approve.